.zenodo.json

```
@@ -92,5 +92,18 @@
     "license": {
         "id": "Apache-2.0"
     },
-    "title": "ESMValCore"
+    "title": "ESMValCore",
+    "communities": [
+        {
+            "identifier": "is-enes3"
+        },
+        {
+            "identifier": "ecfunded"
+        }
```

```suggestion
        {
        {
            "identifier": "dlr_de"
        }
```
This is something I added recently.

---

Added